blueprints/app/files/package.json

```diff
@@ -12,10 +12,12 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
-    "test": "ember test"
+    "test": "npm-run-all lint:* test:*",
```

Comment:
Since this isn't using --parallel, this is like an &&?

Reply:
Yes, it will avoid running tests if linting fails.